dkg/dkg_test.go

```diff
@@ -313,11 +326,16 @@ func verifyDKGResults(t *testing.T, def cluster.Definition, dir string) {
 		require.NoError(t, lock.VerifySignatures())
 		locks = append(locks, lock)
 
-		for _, val := range lock.Validators {
+		for j, val := range lock.Validators {
 			// Assert Deposit Data
 			require.EqualValues(t, val.PubKey, val.DepositData.PubKey)
 			require.EqualValues(t, 32_000_000_000, val.DepositData.Amount)
 
+			if !cluster.SupportPregenRegistrations(lock.Version) {
```

Comment:
can we have similar verification for node signatures as well?

Reply:
yes I added that